learnCSharp/Program.cs

```diff
@@ -27,25 +32,25 @@
                 Commands.ListAllCommands();
                 break;
             case "list":
-                Commands.PrintAllTodos(todoList);
+                Commands.PrintAllTodos(todoRepository);
                 break;
             case "remove":
-                Commands.RemoveTodo(todoList, parsedInput);
+                Commands.RemoveTodo(todoRepository, parsedInput);
                 break;
             case "owner":
                 curOwner = Commands.ChangeUser(parsedInput);
                 break;
             case "new":
-                todoList.Add(Commands.CreateTask(parsedInput, curOwner));
+                await todoRepository.NewTodo(Commands.CreateTask(parsedInput, curOwner));
```

Comment:
Ultimately, I'd move all the todoRepository calls into the commands themselves. This isn't a priority now, but we'll talk about it when we get to step 5!